### PR TITLE
feat: use loader

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
Following the UseTexture component philosophy, I drop this PR.

I have a problem with typescript (since I don't understand TS)  
line11 -  src/composables/useLoader/component.vue

Also (as I normally don't work with the core) I found weird that so many components have been change applying the linter

### CODE REVIEW

**Comment by JaimeTorrealba** [comment]
Can i merge this one? @alvarosabu

File: docs/es/api/composables.md [hunk 1]
```diff
@@ -93,6 +93,33 @@ Dado que el composable `useLoader` devuelve una promesa, puedes usarlo con `asyn
 </template>
 ```
 
+### UseLoader como component
```
Comment (alvarosabu) [inline docs/es/api/composables.md:96]:
```suggestion
### UseLoader como componente
```

**Review by alvarosabu — APPROVED** [review]
Great job as always @JaimeTorrealba

**Comment by alvarosabu** [comment]
@JaimeTorrealba you have some conflicts pending, when you solve them we can merge

**Pull request merged**